src/roomba_robot/main.c

```diff
@@ -1,109 +0,0 @@
-#include <kipr/wombat.h>
-
-/* Defining Boolean values */
-#define false 0
-#define true 1
-#define boolean int
-
-/* Pre-defining functions */
```

Comment:
Keep pre-defined functions and move the main() function to the top again! Define all other functions below the main function

Reply:
we already removed this code for several reasons

Reply:
In that case, state the reasons. I can see that the code got removed, but I don't understand why. Also, I still urge you to keep the pre-defined functions, so that the main function can be the first function in the file.